models/x2y2/ethereum/x2y2_ethereum_events.sql

```diff
@@ -71,9 +74,9 @@ WITH aggregator_routed_x2y2_txs AS (
 , aggregator_routed_x2y2_txs_formatted AS (
     SELECT block_time
     , block_number
-    , e721.to AS buyer
+    , buyer
     , seller
-    , token_id
+    , ROUND(token_id, 0) AS token_id
```

Comment:
Can you please enforce the token_id type to be an int here just to make sure ?

Reply:
Done!